common/metrics/client.go

```diff
@@ -59,6 +59,7 @@ func NewClient(scope tally.Scope, serviceIdx ServiceIdx) Client {
 	for idx, def := range ScopeDefs[Common] {
 		scopeTags := map[string]string{
 			OperationTagName: def.operation,
+			namespace:        namespaceAllValue,
```

Comment:
so this sets the default namespace tag value in case later the value is not set?